IdentityCore/tests/integration/MSIDAuthorityIntegrationTests.m

```diff
@@ -702,7 +703,7 @@ - (void)testDiscoverAuthority_whenValidCloudADFSWithNilResponseAndErrorAuthority
                                                          respondWithError:error];
     NSMutableDictionary *headers = [[MSIDDeviceId deviceId] mutableCopy];
     headers[@"Accept"] = @"application/json";
-    headers[@"x-ms-PkeyAuth"] = @"1.0";
+    headers[kMSIDPKeyAuthHeader] = @"1.0";
```

Comment:
Where is this header value being set for the actual code? I probably missed it

Reply:
In MSIDWorkplaceJoinConstants.m : https://github.com/AzureAD/microsoft-authentication-library-common-for-objc/blob/71cde46ab407baa638def555af17dbd2e993226c/IdentityCore/src/workplacejoin/MSIDWorkPlaceJoinConstants.m#L32 . It is set as x-ms-PKeyAuth or x-ms-PKeyAuth+ according to macro MSID_ENABLE_ECC_SUPPORT